compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -720,7 +722,8 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     ),
     // Ensure the argument to this function is &&str during const-check.
     rustc_attr!(
-        rustc_const_panic_str, Normal, template!(Word), WarnFollowing, INTERNAL_UNSTABLE
+        rustc_const_panic_str, Normal, template!(Word),
+        WarnFollowing, INTERNAL_UNSTABLE
```

Comment:
spurious formatting changes, please revert and rebase

Reply:
> spurious formatting changes, please revert and rebase

Done. Thank you.